projects/ion/src/lib/checkbox/checkbox.component.scss

```
@@ -0,0 +1,121 @@
+@import '../../styles/colors/colors.scss';
```

You should import index, not colors.

```suggestion
@import '../../styles/index.scss';
```